src/main/kotlin/frc/team5499/frc2018Kotlin/utils/Vector2.kt

```diff
@@ -29,10 +29,51 @@ data class Vector2(val x: Double, val y: Double) {
 
     fun distanceTo(other: Vector2) = (this - other).magnitude
 
+    fun getHeading(): Double {
```

Comment:
isnt there an `angle` variable in the vector2 class already? maybe Im wrong. If there isn't, make sure you test this method because it was taken from a different source (Mitchy Boi) so they may not be fully compatible.

Reply:
there is, figure out how to use that in this. idk if we even need the angle for the vector2 tbh. something to look at